seatunnel-connectors-v2/connector-file/connector-file-base/src/main/java/org/apache/seatunnel/connectors/seatunnel/file/source/reader/ExcelReadStrategy.java

```diff
@@ -0,0 +1,200 @@
+import org.apache.seatunnel.common.utils.DateUtils;
+import org.apache.seatunnel.common.utils.TimeUtils;
+import org.apache.seatunnel.connectors.seatunnel.file.config.BaseSourceConfig;
+import org.apache.seatunnel.connectors.seatunnel.file.config.HadoopConf;
+import org.apache.seatunnel.connectors.seatunnel.file.exception.FileConnectorException;
+
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.fs.FSDataInputStream;
+import org.apache.hadoop.fs.FileSystem;
+import org.apache.hadoop.fs.Path;
+import org.apache.poi.ss.usermodel.Cell;
+import org.apache.poi.ss.usermodel.CellType;
+import org.apache.poi.ss.usermodel.DataFormatter;
+import org.apache.poi.ss.usermodel.DateUtil;
+import org.apache.poi.ss.usermodel.Row;
+import org.apache.poi.ss.usermodel.Sheet;
+import org.apache.poi.ss.usermodel.Workbook;
+import org.apache.poi.xssf.usermodel.XSSFWorkbook;
+
+import lombok.SneakyThrows;
+
+import java.math.BigDecimal;
+import java.nio.charset.StandardCharsets;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Map;
+
+import static org.apache.seatunnel.common.utils.DateTimeUtils.Formatter.YYYY_MM_DD_HH_MM_SS;
+
+public class ExcelReadStrategy extends AbstractReadStrategy {
+
+    private final DateUtils.Formatter dateFormat = DateUtils.Formatter.YYYY_MM_DD;
+
+    private final DateTimeUtils.Formatter datetimeFormat = YYYY_MM_DD_HH_MM_SS;
+    private final TimeUtils.Formatter timeFormat = TimeUtils.Formatter.HH_MM_SS;
+
+    @SneakyThrows
+    @Override
+    public void read(String path, Collector<SeaTunnelRow> output) {
+        Configuration conf = getConfiguration();
+        FileSystem fs = FileSystem.get(conf);
+        Map<String, String> partitionsMap = parsePartitionsByPath(path);
+        Path filePath = new Path(path);
+        FSDataInputStream file = fs.open(filePath);
+        Workbook workbook = new XSSFWorkbook(file);
+        Sheet sheet =
+                pluginConfig.hasPath(BaseSourceConfig.SHEET_NAME.key())
+                        ? workbook.getSheet(
+                                pluginConfig.getString(BaseSourceConfig.SHEET_NAME.key()))
+                        : workbook.getSheetAt(0);
+        Row rowTitle = sheet.getRow(0);
+        int cellCount = rowTitle.getPhysicalNumberOfCells();
+        cellCount = partitionsMap.isEmpty() ? cellCount : cellCount + partitionsMap.size();
+        SeaTunnelRow seaTunnelRow = new SeaTunnelRow(cellCount);
+        SeaTunnelDataType<?>[] fieldTypes = seaTunnelRowType.getFieldTypes();
+        int rowCount = sheet.getPhysicalNumberOfRows();
+        for (int i = 1; i < rowCount; i++) {
+            Row rowData = sheet.getRow(i);
+            if (rowData != null) {
+                for (int j = 0; j < cellCount; j++) {
+                    Cell cell = rowData.getCell(j);
+                    if (cell != null) {
```

Comment:
if cell == null, field should set null, BTW, should support column projection. `read_columns` can controll this feature.